pkg/alexa/gen/model.go

```diff
@@ -40,8 +40,10 @@ func (m *ModelBuilder) WithDelegationStrategy(strategy string) *ModelBuilder {
 
 func (m *ModelBuilder) AddLocale(locale string, invocation string) *ModelBuilder {
 	loc := l10n.NewLocale(locale)
-	m.registry.Register(loc)
-	loc.TextSnippets[m.invocation] = []string{invocation}
+	if err := m.registry.Register(loc); err != nil {
+		return nil
```

Comment:
This will cause a panic if you use the builder, as you have no idea that `nil` is being returned.